### PR TITLE
Help Center: Add articles feedback form

### PR DESCRIPTION
**Pull request by renancarvalho**
Related to #86531

## Proposed Changes

* This PR adds a small feedback form at the end of articles. More context #86531

image

## Testing Instructions

* Use live link
* Navigate to any article in Help Center and scroll all the way down
* You should see the feedback 👍 or 👎 
* After selecting “yes” or “no”, the users should be able to optionally submit any free text
* With Track Vigilante (PCYsg-T9m-p2), check if the event `calypso_inlinehelp_article_feedback_click` is being logged
* Also Make sure to check atomic sites
* Results can be found in Crowd Signal (https://app.crowdsignal.com/), search for `HelpCenter articles feedback`
* The after submitting animation is from Crowd Signal, we can provide a custom message, will investigate what is the best.

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-aUh-p2)?

### CODE REVIEW

File: packages/help-center/src/components/help-center-feedback-form.tsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+					{ __( 'No' ) } <ThumbsDownIcon />
+				</button>
+			</div>
+		</>
+	);
+
+	const FeedbackTextArea = () => (
+		<>
+			<p>{ __( 'How we can improve?' ) }</p>
+			<iframe
```
Comment (renancarvalho) [inline packages/help-center/src/components/help-center-feedback-form.tsx:48]: Loads `crowdsignal` survey, the submit happens in this tool and not in HelpCenter. To make sure we save the right data, we pass post and blog ID and also which choice the user selected before, yes or no.

File: packages/help-center/src/components/help-center-feedback-form.tsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+			<p>{ __( 'Did you find the answer to your question?' ) }</p>
+			<div className="help-center-feedback-form__buttons">
+				<button
+					// 1 is used as `yes` in crowdsignal as well, do not change
+					onClick={ () => handleFeedbackClick( 1 ) }
+				>
+					{ __( 'Yes' ) } <ThumbsUpIcon />
+				</button>
+				<button
+					// 2 is used as `no` in crowdsignal as well, do no change
```
Comment (agrullon95) [inline packages/help-center/src/components/help-center-feedback-form.tsx]: Small change here to "Do not change"

File: packages/help-center/src/components/help-center-feedback-form.tsx [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+	const FeedbackButtons = () => (
+		<>
+			<p>{ __( 'Did you find the answer to your question?' ) }</p>
+			<div className="help-center-feedback-form__buttons">
+				<button
+					// 1 is used as `yes` in crowdsignal as well, do not change
+					onClick={ () => handleFeedbackClick( 1 ) }
+				>
+					{ __( 'Yes' ) } <ThumbsUpIcon />
```
Comment (agrullon95) [inline packages/help-center/src/components/help-center-feedback-form.tsx:33]: Let's update this to use thumbs-up emoji to match the /support.

File: packages/help-center/src/components/help-center-feedback-form.tsx [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+				</button>
+				<button
+					// 2 is used as `no` in crowdsignal as well, do not change
+					onClick={ () => handleFeedbackClick( 2 ) }
+				>
+					{ __( 'No' ) } <ThumbsDownIcon />
```
Comment (agrullon95) [inline packages/help-center/src/components/help-center-feedback-form.tsx:39]: Let's update this to use thumbs-down emoji to match the /support.

File: packages/help-center/src/components/help-center-feedback-form.tsx [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+				</button>
+			</div>
+		</>
+	);
+
+	const FeedbackTextArea = () => (
+		<>
+			<p>{ __( 'How we can improve?' ) }</p>
+			<iframe
+				title="Feedback Form"
```
Comment (agrullon95) [inline packages/help-center/src/components/help-center-feedback-form.tsx]: Lets use the translation function here.

**Review by agrullon95 — CHANGES_REQUESTED** [review]
@renancarvalho - Nice work!

We should update the icons to emojis (Edit: This can be updated in another PR) and add the translation function to the iframe title.
The feedback survey is triggering the tracks event and tracking in crowdsignal correctly.

**Comment by renancarvalho** [comment]
@agrullon95 thank you for reviewing, regarding the 👍 and 👎 (emoji or icon), I got these icons from Calypso itself, I think it makes more sense to match Calypso "style", WDYT?

**Comment by agrullon95** [comment]
> @agrullon95 thank you for reviewing, regarding the 👍 and 👎 (emoji or icon), I got these icons from Calypso itself, I think it makes more sense to match Calypso "style", WDYT?

Let's keep it as is. I misread the slack conversation regarding these icons/emojis.

**Review by agrullon95 — APPROVED** [review]
The PR LGTM now. I tested on Simple and Atomic. We can wait for translation before merging.

**Pull request merged**